### PR TITLE
Adds the name of the occupant to the examine text of sleepers and body scanners 

### PR DESCRIPTION
**Pull request by Divulf**
## What Does This PR Do
Adds the name of the occupant to the examine text of sleepers and body scanners.   The new description also tells you if the occupant inside is dead or not.  Concealed identities will remain concealed.

## Images of changes
sleeper1
sleeper2

## Testing
Stuffed Pun Pun into sleeper and body scanner while both alive and dead.

## Changelog
:cl:
add: Name of the occupant to the examine text of sleepers and body scanners 
/:cl:

### CODE REVIEW

File: code/game/machinery/Sleeper.dm [hunk 1]
```diff
@@ -33,6 +33,11 @@
 
 /obj/machinery/sleeper/examine(mob/user)
 	. = ..()
+	if(occupant)
+		if(occupant.is_dead())
+			. += "<span class='warning'>You see [occupant.name] inside. [occupant.p_they(TRUE, FALSE)] is dead!</span>"
+		else
+			. += "You see [occupant.name] inside."
```
Comment (Contrabang) [inline code/game/machinery/Sleeper.dm:40]: Could you use `<span class='notice'>` here to differentiate this from the rest of the examine text?

File: code/game/machinery/adv_med.dm [hunk 2]
```diff
@@ -14,6 +14,11 @@
 
 /obj/machinery/bodyscanner/examine(mob/user)
 	. = ..()
+	if(occupant)
+		if(occupant.is_dead())
+			. += "<span class='warning'>You see [occupant.name] inside. [occupant.p_they(TRUE, FALSE)] is dead!</span>"
+		else
+			. += "You see [occupant.name] inside."
```
Comment (Contrabang) [inline code/game/machinery/adv_med.dm:21]: same here

**Review by Contrabang — CHANGES_REQUESTED** [review]
Good idea

**Comment by Divulf** [comment]
cryo-cell

**Comment by Denthamos** [comment]
Does it only show the name of the person if their face and/or ID is present and visible?  Does it show a persons name despite having a full mask on and no ID?

**Comment by Divulf** [comment]
> Does it only show the name of the person if their face and/or ID is present and visible? Does it show a persons name despite having a full mask on and no ID?

It uses their apparent name, same as examining the subject normally.

File: code/game/machinery/Sleeper.dm [hunk 3]
```diff
@@ -33,6 +33,11 @@
 
 /obj/machinery/sleeper/examine(mob/user)
 	. = ..()
+	if(occupant)
+		if(occupant.is_dead())
+			. += "<span class='warning'>You see [occupant.name] inside. [occupant.p_they(TRUE, FALSE)] is dead!</span>"
```
Comment (lewcc) [inline code/game/machinery/Sleeper.dm:38]: I did some diving, and `occupant.name` itself is indeed updated on human life, so we thankfully don't need to check if they're hidden or not.
Only other thing is that I'm not sure why you're passing `FALSE` in for the temp gender (second argument). For humans, it gets overridden by `get_visible_gender()` anyway, so I'm pretty sure it's unnecessary.

Anyway, please make these sorts of changes through all the instances in the PR.
```suggestion
			. += "<span class='warning'>You see [occupant.name] inside. [occupant.p_they(TRUE)] [occupant.p_are()] dead!</span>"
```

File: code/game/machinery/adv_med.dm [hunk 4]
```diff
@@ -14,6 +14,11 @@
 
 /obj/machinery/bodyscanner/examine(mob/user)
 	. = ..()
+	if(occupant)
+		if(occupant.is_dead())
+			. += "<span class='warning'>You see [occupant.name] inside. [occupant.p_they(TRUE, FALSE)] is dead!</span>"
```
Comment (lewcc) [inline code/game/machinery/adv_med.dm:19]:
```suggestion
			. += "<span class='warning'>You see [occupant.name] inside. [occupant.p_they(TRUE)] [occupant.p_are()] dead!</span>"
```

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 5]
```diff
@@ -31,6 +31,11 @@
 
 /obj/machinery/atmospherics/unary/cryo_cell/examine(mob/user)
 	. = ..()
+	if(occupant)
+		if(occupant.is_dead())
+			. += "<span class='warning'>You see [occupant.name] inside. [occupant.p_they(TRUE, FALSE)] is dead!</span>"
```
Comment (lewcc) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:36]:
```suggestion
			. += "<span class='warning'>You see [occupant.name] inside. [occupant.p_they(TRUE)] [occupant.p_are()] dead!</span>"
```

**Review by lewcc — COMMENTED** [review]
Denth si

**Comment by Divulf** [comment]
are

**Review by S34NW — APPROVED** [review]

**Pull request merged**